app/src/androidTest/java/org/mozilla/tv/firefox/ui/IWebViewExecuteJavascriptTest.kt

```diff
@@ -44,7 +45,7 @@ class IWebViewExecuteJavascriptTest {
 
         // Load the mock page.
         onView(withId(R.id.navUrlInput))
-                .perform(typeText(url), pressImeActionButton())
+                .perform(typeText(url), closeSoftKeyboard(), pressImeActionButton())
```

Comment:
Are these changes related to this PR? If not, can we take them out? If so, can you explain how it's related? This PR is challenging to read already.

Reply:
These tests were failing on CI.  I think we've seen some periodic failures from them in the past, so rerunning might solve it, but fixing it seemed better.  I can pull them out and throw them into their own PR.